src/styles/public/single.scss

```diff
@@ -0,0 +1,10 @@
+/* stylelint-disable no-descending-specificity */
+
+/* Single Post Styling */
+body.single.single-post {
+
+	.wp-block-group.bcgov-body-content {
+		min-height: 100vh;
+		min-height: 100dvh;
+	}
```

Comment:
according to https://dev.to/frehner/css-vh-dvh-lvh-svh-and-vw-units-27k4 

> The sizes of the dynamic viewport-percentage units are not stable even while the viewport itself is unchanged. Using these units can cause content to resize e.g. while the user scrolls the page. Depending on usage, this can be disturbing to the user and/or costly in terms of performance.

Is there some way to avoid using 'dvh' here?

Reply:
Dvh is used in several places within this repo. I'm not opposed to researching and reviewing the usage, but I think that would be a separate ticket.

Reply:
That article is a couple/few years old now. dvh is recommended for setting height as a mobile viewport is often dynamic when scrolling. As a unit it has very good coverage these days. 
<img width="1397" alt="image" src="https://github.com/bcgov/bcgov-wordpress-block-theme/assets/5200460/1debdfbd-263a-4574-9ef0-3d6c657eacfe">